src/verilog/convert_literals.cpp

```diff
@@ -132,6 +132,24 @@ constant_exprt convert_integral_literal(const irep_idt &value)
       rest += tolower(ch);
   }
 
+  // special case the "unbased unsized literals"
+  if(rest == "'0")
+  {
+    return from_integer(0, unsignedbv_typet{1});
+  }
+  else if(rest == "'1")
+  {
+    return from_integer(0, unsignedbv_typet{1});
+  }
```

Comment:
Should this be `if(rest == "'0" || rest == "'1")` to make clear there is no copy&paste mistake here?